drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -1175,6 +1215,10 @@ bnxt_dev_init(struct rte_eth_dev *eth_dev)
 				goto error_free;
 			}
 		}
+		bp->pf->vf_info = rte_malloc("bnxt_vf_info",
```

Comment:
bp->pf should be a struct, not a pointer.
bp->pf.vf_info should already be allocated here.  This looks like a memory leak.